### PR TITLE
Update to latest versions.

### PR DESCRIPTION
**Pull request by BrandonShutter**
I've tested locally, and everything still works.

### CODE REVIEW

File: ansible/Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# -*- mode: ruby -*-
```
Comment (swalkinshaw) [inline ansible/Vagrantfile]: This file needs to exist in the root. I think the only change is to `ANSIBLE_PATH`.

File: ansible/group_vars/development [hunk 2]
```diff
@@ -20,12 +20,12 @@ wordpress_sites:
...
-      wp_siteurl: http://roots-example-project.dev/wp
+      wp_home: http://example.dev
+      wp_siteurl: http://example.dev/wp
       wp_env: development
-      db_name: roots_example_project_dev
-      db_user: roots_example_project_dbuser
-      db_password: roots_example_project_dbpassword
+      db_name: example_dev
+      db_user: example_dbuser
+      db_password: example_dbpassword
```
Comment (swalkinshaw) [inline ansible/group_vars/development]: All of the existing settings need to remain here. I think the only actual change was "project" to "site" on L9.

File: ansible/group_vars/production [hunk 3]
```diff
@@ -1,4 +1,4 @@
-mysql_root_password: B3LkKUpcZVx4bpLXKXpiez%R
+mysql_root_password: productionpw
```
Comment (swalkinshaw) [inline ansible/group_vars/production]: Same as above but for this file.

File: ansible/hosts/production [hunk 4]
```diff
@@ -1,5 +1,5 @@
 [web]
-104.131.9.84
+192.168.50.5
```
Comment (swalkinshaw) [inline ansible/hosts/production]: :scissors:

**Comment by swalkinshaw** [comment]
@BrandonShutter thanks for this. I made some comments. Just be careful when updating the group_vars and hosts files.

**Comment by BrandonShutter** [comment]
Updating things now, I’ll submit a PR in a few moments.  

## 

Brandon Shutter

**Comment by BrandonShutter** [comment]
Updated vars.

File: ansible/group_vars/production [hunk 5]
```diff
@@ -27,7 +27,7 @@ wordpress_sites:
       - roots-example-project.com
     local_path: '../site' # path targeting local Bedrock project directory (relative to Ansible root)
     repo: git@github.com:roots/bedrock.git
-    subtree: site
```
Comment (swalkinshaw) [inline ansible/group_vars/production:30]: `subtree: site` needs to remain as well

**Comment by swalkinshaw** [comment]
Can you squash commits once the other fix is done? Then it should all be good :+1:

**Comment by BrandonShutter** [comment]
Boom :)

**Comment by swalkinshaw** [comment]
Thanks!

**Pull request merged**